Code/JavaWrappers/ChemTransforms.i

```diff
@@ -101,7 +107,11 @@ RDKit::ROMol * new_molzip(
 			  const RDKit::ROMol &a,
 			  const RDKit::MolzipParams &params=RDKit::MolzipParams());
 
-          
+
+RDKit::ROMol * new_molzip(
```

Comment:
do you need a `%newobject` for this to tell the wrappers that they own the results?

Reply:
It's already there at line 90 (this `new_molzip` is an overload of existing functions).

Reply:
ah, sorry, that didn't show up in the PR view.